Update-Dell7450Firmware.ps1

```
@@ -0,0 +1,544 @@
+                $updateVariants += $drive.Variant
+            }
+        } elseif ($comparison -eq 0) {
+            Write-Host "Status: FIRMWARE CURRENT" -ForegroundColor Green
+        } else {
+            Write-Host "Status: FIRMWARE NEWER THAN TARGET" -ForegroundColor Green
+        }
+    }
+
+    # Exit if WI found (manual intervention needed)
+    if ($hasWI) {
+        Write-Host "`n*** Script cannot continue - WI drives require manual firmware installation ***" -ForegroundColor Red
```

PSScriptAnalyzer (1.24.0)

[warning] File 'Update-Dell7450Firmware.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)